src/view/element.js

```diff
@@ -674,6 +674,37 @@ export default class Element extends Node {
 	}
 
 	/**
+	 * Returns identity string based on element's name, styles, classes and other attributes.
+	 * Two elements that {@link #isSimilar are similar} will have same identity string.
+	 * It has the following format:
+	 *
+	 *		"name|classes(class1,class2,class3)|styles(style1=val1,style2=val2)|attributes(attr1=val1,attr2=val2)"
```

Comment:
I don't like the fact that we introduce one more custom format. Can't we return it in HTML-like format:
`name classes="class1,class2,class3" styles="style1:val1;style2:val2" attr1="val1" attr2="val2"`. It does not seams to be longer and people are used to parse it ealiy while debugging.